### PR TITLE
removed all references to paas

### PR DESCRIPTION
**Pull request by temitope777**
### Trello card
https://trello.com/c/9BSGwVYz/841-code-clean-up-school-experience
### Context
To remove all references to deprecated paas codes and references and remove redundant prefixes and suffixes
### Changes proposed in this pull request
removed all references to paas and all aks  prefixes and suffixes

### CODE REVIEW

**Comment by temitope777** [comment]
[Code clean up - School experience](https://trello.com/c/9BSGwVYz/841-code-clean-up-school-experience)

File: .github/workflows/actions/deploy/action.yml [hunk 1]
```diff
@@ -1,146 +1,78 @@
...
+        IMAGE_TAG: ${{ steps.sha.outputs.short }}
 
-    - name: Smoke tests
+    - name: Retrieve DSI hostname
+      id: dsi_hostname
       shell: bash
       run: |
-        if [ "${{steps.variables.outputs.RUN_TEST}}" == "true" ]
-        then
-               tests/confidence/healthcheck.sh  "${{steps.variables.outputs.healthcheck}}"  "${{ steps.sha.outputs.short }}"
```
Comment (saliceti) [inline .github/workflows/actions/deploy/action.yml:129]: This smoke test is useful. Can we make it work or create a new card to add it back?

**Comment by saliceti** [comment]
Please search for "paas" in the repo. I can see more references to it.

File: .github/workflows/actions/deploy/action.yml (hunk 1 above)
Comment (temitope777) [reply .github/workflows/actions/deploy/action.yml:129]: A new card created for smoke test

File: .github/dependabot.yml [hunk 2]
```diff
@@ -36,8 +36,3 @@ updates:
     directory: /
     schedule:
       interval: daily
```
Comment (RMcVelia) [inline .github/dependabot.yml:38]: Just removed /terraform/paas/ but why not change to /terraform/aks/

File: docs/documents/DevOps/workflows.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ layout: page
 # Workflows
 
 ## Overview
-Workflows are orchestrated via [github actions](https://docs.github.com/en/actions) which build, test and deliver our code to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/).
+Workflows are orchestrated via [github actions](https://docs.github.com/en/actions) which build, test and deliver our code to the [GOV.UK](https://www.cloud.service.gov.uk/).
```
Comment (RMcVelia) [inline docs/documents/DevOps/workflows.md:11]: deliver our code to AKS.

File: docs/documents/DevOps/workflows.md [hunk 4]
```diff
@@ -8,11 +8,11 @@ layout: page
...
 
 Guidance about the use of Github Actions can be found on the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/automation/github-actions) page.
 
-Delivery to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/) is via [Terraform](https://technical-guidance.education.gov.uk/infrastructure/hosting/govuk-paas/tooling/#terraform) which delivers [containers](https://technical-guidance.education.gov.uk/infrastructure/hosting/govuk-paas/tooling/#docker) built by the workflow and stored in [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)(GHCR)
+Delivery to the [GOV.UK](https://www.cloud.service.gov.uk/) is via Terraform which delivers containers built by the workflow and stored in [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)(GHCR)
```
Comment (RMcVelia) [inline docs/documents/DevOps/workflows.md:15]: Delivery to AKS is via Terraform

File: terraform/aks/config/production_Terrafile [hunk 5]
```diff
@@ -1,3 +1,3 @@
 aks:
     source:  "https://github.com/DFE-Digital/terraform-modules"
-    version: "stable"
+    version: "v0.21.0-alpha"
```
Comment (RMcVelia) [inline terraform/aks/config/production_Terrafile:3]: change back to stable?

Comment (temitope777) [reply terraform/aks/config/production_Terrafile:3]: this was done in another PR

File: .github/dependabot.yml (hunk 2 above)
Comment (temitope777) [reply .github/dependabot.yml:38]: done

File: docs/documents/DevOps/workflows.md (hunk 3 above)
Comment (temitope777) [reply docs/documents/DevOps/workflows.md:11]: added

File: terraform/aks/config/production_Terrafile (hunk 5 above)
Comment (RMcVelia) [reply terraform/aks/config/production_Terrafile:3]: well won't this PR change it back to v0.21.0-alpha unless you remove it

File: docs/documents/DevOps/workflows.md (hunk 4 above)
Comment (RMcVelia) [reply docs/documents/DevOps/workflows.md]: I would remove '[GOV.UK](https://www.cloud.service.gov.uk/)'

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**